Revision to neonatal intervention docs (#1598)
ihmeuw/vivarium_research: docs/source/models/intervention_models/neonatal/antibiotics_intervention.rst

.. math::

   \text{CSMR}_i^\text{sepsis} = \text{CSMR}^\text{sepsis}_{\text{BW}_i, \text{GA}_i} \cdot (1 - \text{PAF}_\text{no antibiotics}) \cdot \text{RR}_i^\text{no antibiotics}

where :math:`\text{RR}_i^\text{no antibiotics}` is simulant *i*'s individual relative risk for "no antibiotics", meaning :math:`\text{RR}_i^\text{no antibiotics} = \text{RR}_\text{no antibiotics}` if simulant *i* accesses a facility without antibiotics, and :math:`\text{RR}_i^\text{no antibiotics} = 1` if simulant *i* accesses a facility *with* antibiotics.

If there are other interventions also affecting the CSMR of sepsis, the pipeline will combine these effects, and we can write out the math for this risk explicitly as 

.. math::

   \text{CSMR}^\text{sepsis}_{i, \text{updated}} = \text{CSMR}^\text{sepsis}_{i, \text{original}} \cdot (1 - \text{PAF}_\text{no antibiotics}) \cdot \text{RR}_i^\text{no antibiotics}

This reduces to the previous formula if there are no other interventions, and we would have 

.. math::

   \text{CSMR}^\text{sepsis}_{i, \text{original}} = \text{CSMR}^\text{sepsis}_{\text{BW}_i, \text{GA}_i}

> While we are searching the literature for an appropriate value for the relative risk, we will use a stand-in value with an origin I have failed to record.

Reviewer: Maybe someone else remembers?

        p(\text{sepsis}) 
        &= \sum_{\text{paths without antibiotics}} p(\text{path})\cdot p(\text{sepsis}|\text{no antibiotics})\\
        &+ \sum_{\text{paths with antibiotics}} p(\text{path})\cdot p(\text{sepsis}|\text{antibiotics})\\[.1in]
        p(\text{sepsis}|\text{no antibiotics}) &= \text{RR}_\text{no antibiotics} \cdot p(\text{sepsis}|\text{antibiotics})
    \end{align*}

where :math:`p(\text{sepsis})` is the probability of dying from sepsis in the general population, and :math:`p(\text{sepsis}|\text{antibiotics})` and :math:`p(\text{sepsis}|\text{no antibiotics})` are the probability of dying from sepsis in setting with and without access to antibiotics.  For each path through the decision tree, :math:`p(\text{path})` is the probability of that path; for example the path that includes the edges labeled BEmONC and unavailable occurs with probability that the birth is in a BEmONC facility times the probability that the facility has antibiotics available.

> When we fill in the location-specific values for delivery facility rates, antibiotics coverage, relative risk of mortality with antibiotics access, and mortality probability (which is also age-specific), this becomes a system of two linear equations have two unknown (p_sepsis_w and p_sepsis_wo), which we can solve analytically using the same approach as in the :ref:`cpap calibration <cpap_calibration>`.

Reviewer: ```suggestion
When we fill in the location-specific values for delivery facility rates, antibiotics coverage, relative risk of mortality with antibiotics access, and mortality probability (which is also age-specific), this becomes a system of two linear equations with two unknowns (:math:`p(\text{sepsis}|\text{antibiotics})` and :math:`p(\text{sepsis}|\text{no antibiotics})`), which we can solve analytically using the same approach as in the :ref:`cpap calibration <cpap_calibration>`.
```

**Alternative PAF Derivation**: An alternative, and possibly simpler derivation of the PAF that will calibrate this model comes from the observation that :math:`\text{PAF} = 1 - \frac{1}{\mathbb{E}(\text{RR})}`.  If we define 

.. math::

   p(\text{no antibiotics}) = \sum_{\text{paths without antibiotics}} p(\text{path}),

then can use this to expand the identity

.. math::

   \text{PAF}_\text{no antibiotics} = 1 - \frac{1}{\mathbb{E}(\text{RR})}.

Since our risk exposure has two categories,

.. math::

>    \mathbb{E}(\text{RR}) = p(\text{no antibiotics}) \cdot \text{RR}_\text{no antibiotics} + (1 - p(\text{no antibiotics})) \cdot 1.

Reviewer: Is there a reason not to make this the primary explanation on both pages? The way this is written, it seems to imply that this is an option for antibiotics but *not* for CPAP, but it seems like it would apply equally to either.

Author: Jim already implementing things for CPAP, so I thought I'd leave it the way it was when he implemented, at least for now.

Reviewer: Makes sense. Can we add this alternative to both pages then, or explicitly note that it can work for either? Like I said I was a bit thrown off by its placement/its absence on the other page.

Author: I'm fine with a explicit note and a link.  Do you have time to add it?

---------------------------

- We assume that antibiotics availability captures actual use, and not simply the treatment being in the facility 

> - We assume that the delivery facility is also the facility where a sick neonatal will seek care for sepsis

Reviewer: ```suggestion
- We assume that the delivery facility is also the facility where a sick neonate will seek care for sepsis
```

- We assume that the relative risk of sepsis mortality with antibiotics in practice is a value that we can find in the literature
- We have excluded the effect of antibiotics on pneumonia mortality, because this cause is currently lumped with 'other causes'

Validation and Verification Criteria
------------------------------------

- Population-level mortality rate should be the same as when this intervention is not included in the model

> - The ratio of sepsis deaths per birth among those with antibiotics access divided by those without antibiotics access should equal the relative risk parameter used in the model

Reviewer: I don't understand this change.

Author: I switched things above to be in terms of the relative risk of *not* having access to antibiotics, and this inverts the ratio to match that

Reviewer: I got that but I still think you've got it backwards

Author: oh, you are right!
```suggestion
- The ratio of sepsis deaths per birth among those without antibiotics access divided by those with antibiotics access should equal the relative risk parameter used in the model
```
